packages/lambdas/src/sqs-to-fhir.ts

```diff
@@ -167,21 +162,13 @@ export async function handler(event: SQSEvent) {
       processFHIRResponse(response, event, log);
 
       await cloudWatchUtils.reportMetrics(metrics);
-      await ossApi.notifyApi({ ...lambdaParams, status: "success" }, log);
```

Comment:
why is this being removed?

Reply:
From the PR's description:

> It also moves the notification about completed CDA>FHIR conversion from the FHIR server lambda to the Converter lambda.

The reason is because we don't need to wait for the bundle to be inserted on FHIR server anymore, since the consolidated is being returned from the consolidated bundle.

But this made me realize we should wait to make this move until all cxs are migrated to get consolidated from S3, otherwise those not migrated yet might get incomplete bundles on the WH.

I'll revert this and move it to its own PR.

Reply:
reintroduced